cli/slsa-verifier/main.go

```diff
@@ -178,6 +179,32 @@ func getArtifactHash(artifactImage, artifactPath string) (string, error) {
 		}
 		return hex.EncodeToString(h.Sum(nil)), nil
 	}
-	// Retrieve image digest
-	return container.GetImageDigest(artifactImage)
+	// Retrieve the image digest.
+	digest, err := container.GetImageDigest(artifactImage)
+	if err != nil {
+		return "", fmt.Errorf("%w: %s", serrors.ErrorImageHash, err)
+	}
+
+	// Check if the image refers to a file.
```

Comment:
Right now local files are only supported in test anyway right? Not opposed here though to including

Reply:
correct. I need this code to allow tests to pass.